modules/update-using-custom-machine-config-pools-unpause.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * updating/update-using-custom-machine-config-pools.adoc
+
+[id="update-using-custom-machine-config-pools-unpause_{context}"]
+= Unpausing the machine config pools
+
+In this canary rollout update process, after the {product-title} update is complete, unpause your custom MCPs one-by-one. Unpausing an MCP allows the Machine Config Operator (MCO) from updating the nodes associated with that MCP.
+
+To pause an MCP:
+
+. Patch the MCP that you want paused:
++
+[source,terminal]
+----
+$ oc patch mcp/<mcp_name> --patch '{"spec":{"paused":false}}' --type=merge
+----
++
+For example:
++
+[source,terminal]
+----
+$  oc patch mcp/workerpool-canary --patch '{"spec":{"paused":false}}' --type=merge
+----
++
+.Example output
++
+[source,terminal]
+----
+machineconfigpool.machineconfiguration.openshift.io/workerpool-canary patched
+----
++
+You can check the progress of the update by using the `oc get machineconfigpools` command.
+
+. Test your applications on the updated nodes to ensure that they are working as expected.
+
+[id="update-using-custom-machine-config-pools-fail_{context}"]
+== In case of application failure
+
+In the event that any of your applications fail on an updated node, keep all the MCPs paused and wait for another {product-title} version with the bug fix and start the update process again.
```

Comment:
This text should be updated with `In case of a failure, such as your applications not working on the updated nodes, you can cordon and drain the nodes in the pool, which moves the application pods to other nodes to help maintain the quality-of-service for the applications. This first MCP should be no larger than the excess capacity.`